### PR TITLE
Insert null checks for context to avoid NPEs

### PR DESCRIPTION
**Pull request by jim-krueger**

### CODE REVIEW

**Review by jkoehler22 — APPROVED** [review]
LGTM

**Comment by jim-krueger** [comment]
#build

**Comment by jim-krueger** [comment]
#run-libby-bot

**Review by andymc12 — COMMENTED** [review]
Copyright headers needs to keep the original date.  It would also be good to have a test - even if it is just a unit test to verify that things work as expected with null servlet requests/responses.  Otherwise, looks good.

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxrs.3.2/src/com/ibm/ws/jaxrs20/injection/HttpServletRequestInjectionProxy.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2018 IBM Corporation and others.
+ * Copyright (c) 2020 IBM Corporation and others.
```
[the remark this answers is not in the document]
Comment (jim-krueger) [reply dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxrs.3.2/src/com/ibm/ws/jaxrs20/injection/HttpServletRequestInjectionProxy.java]: Done

**Comment by jim-krueger** [comment]
#run-libby-bot

**Comment by jim-krueger** [comment]
#build

**Comment by jim-krueger** [comment]
#15550

**Comment by jim-krueger** [comment]
This fix has been successfully tested by the customer who reported the problem.
